games/scid/Portfile

```diff
@@ -13,12 +13,15 @@ long_description        Scid is an open source application to view, edit, and \
                         manage collections of chess games.
 homepage                http://scid.sourceforge.net/
 master_sites            sourceforge
-checksums               rmd160  7245036f6be164955c8fedf0dd22b5f67f37c2da \
-                        sha256  4fde535051786c0f8b13e6a2a20b306e7ba29e7af7394f5e510aae4f87e223e6 \
-                        size    12248886
+checksums               rmd160  2770906e46c4459b837b33cce59489fab223fa76 \
+                        sha256  2ed25781ec3c82d60fcee85259c19fd8934feae2547f9464304cdb01960f86da \
+                        size    15782104
 
+distname                ${name}-code-${version}
 use_zip                 yes
-depends_lib             port:tcl port:tcllib port:zlib port:tDOM port:tk port:tklib
+depends_lib             port:tcl \
+                        port:tDOM \
+                        port:tk
```

Comment:
The port appears to bundle and install its tklib dependency, and so would not appear to need MacPorts' tklib or tcllib. Is this correct?

Reply:
`tklib` and `zlib` can be removed from the dependencies, but `tcllib` is still needed according to [Scid's maintainer](https://sourceforge.net/p/scid/mailman/scid-users/thread/196134929.403481.1597084154250%40mail1.libero.it/#msg37081921).

Reply:
Thanks for checking with scid maintainers. However their response did not clarify for me whether there is actually a tcllib dependency. I have posted to mailing list: https://sourceforge.net/p/scid/mailman/message/37082927/